packages/prisma/schema.prisma

```diff
@@ -359,55 +359,59 @@ model Profile {
-  parent                Team?                   @relation("organization", fields: [parentId], references: [id], onDelete: Cascade)
-  children              Team[]                  @relation("organization")
-  orgUsers              User[]                  @relation("scope")
-  inviteTokens          VerificationToken[]
-  webhooks              Webhook[]
-  timeFormat            Int?
-  timeZone              String                  @default("Europe/London")
-  weekStart             String                  @default("Sunday")
-  routingForms          App_RoutingForms_Form[]
-  apiKeys               ApiKey[]
-  credentials           Credential[]
-  accessCodes           AccessCode[]
-  isOrganization        Boolean                 @default(false)
-  organizationSettings  OrganizationSettings?
-  instantMeetingTokens  InstantMeetingToken[]
-  orgProfiles           Profile[]
-  pendingPayment        Boolean                 @default(false)
-  dsyncTeamGroupMapping DSyncTeamGroupMapping[]
-  isPlatform            Boolean                 @default(false)
-  platformOAuthClient   PlatformOAuthClient[]
-  smsLockState          SMSLockState            @default(UNLOCKED)
-  platformBilling       PlatformBilling?
-  activeOrgWorkflows    WorkflowsOnTeams[]
+  metadata               Json?
+  theme                  String?
+  brandColor             String?
+  darkBrandColor         String?
+  verifiedNumbers        VerifiedNumber[]
+  verifiedEmails         VerifiedEmail[]
+  bannerUrl              String?
+  parentId               Int?
+  parent                 Team?                   @relation("organization", fields: [parentId], references: [id], onDelete: Cascade)
+  children               Team[]                  @relation("organization")
+  orgUsers               User[]                  @relation("scope")
+  inviteTokens           VerificationToken[]
+  webhooks               Webhook[]
+  timeFormat             Int?
+  timeZone               String                  @default("Europe/London")
+  weekStart              String                  @default("Sunday")
+  routingForms           App_RoutingForms_Form[]
+  apiKeys                ApiKey[]
+  credentials            Credential[]
+  accessCodes            AccessCode[]
+  isOrganization         Boolean                 @default(false)
+  organizationSettings   OrganizationSettings?
+  instantMeetingTokens   InstantMeetingToken[]
+  orgProfiles            Profile[]
+  pendingPayment         Boolean                 @default(false)
+  dsyncTeamGroupMapping  DSyncTeamGroupMapping[]
+  isPlatform             Boolean                 @default(false)
+  // Organization's OAuth clients. Organization has them but a team does not.
+  platformOAuthClient    PlatformOAuthClient[]
+  // OAuth client used to create team of an organization. Team has it but organization does not.
+  createdByOAuthClient   PlatformOAuthClient?    @relation("CreatedByOAuthClient", fields: [createdByOAuthClientId], references: [id])
```

Comment:
onDelete: Cascade makes sense here I think, no need to keep the team when the oauth client is deleted

Reply:
Agreed! Implemented in [refactor: delete platform team when oauth client deleted](https://github.com/calcom/cal.com/pull/15961/commits/3952104999b3c04c13f78998a6f604fc62cbbe2e)